### BIG differences, this is a diff between CLUS "Function ATOM" entry and topic selection in CLHS http://clhs.lisp.se/Body/a_atom.htm
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:functions:atom.txt	2017-04-27 22:03:35.000000000 +0100
+++ saved-pages-clus/cl:functions:atom.txt	2017-04-27 22:03:35.000000000 +0100
@@ -1,2 +1,1 @@
-ATOM
-Please select which reference to ATOM you intended
+Function ATOM

-FunctionTypeGlossary Entry
\ No newline at end of file
+Syntax
+ atom object → generalized-boolean
+Arguments and values
+ object - an object.
+ generalized-boolean - a genralized boolean.
+Description
+Returns true if object is of type atom; otherwise, returns false.
+Examples
+(atom 'sss)
+→
+true
+(atom (cons 1 2)
+→
+false
+(atom nil)
+→
+true
+(atom '())
+→
+true
+(atom 3)
+→
+true
+Side Effects
+None.
+Affected By
+None.
+Exceptional Situations
+None.
+See also
+Function LISTP
+Notes
+(atom object)
+  ≡ (typep object 'atom)
+  ≡ (not (consp object))
+  ≡ (not (typep object 'cons))
+  ≡ (typep object '(not cons))
\ No newline at end of file
